client/main.lua

```diff
@@ -60,10 +60,16 @@ Citizen.CreateThread(function()
                     if IsControlJustPressed(0, 38) then
                         QBCore.Functions.TriggerCallback('qb-garage:server:checkVehicleOwner', function(owned)
                             if owned then
-                                TriggerServerEvent('qb-occasions:server:sellVehicleBack', sellVehData)
-                                QBCore.Functions.DeleteVehicle(GetVehiclePedIsIn(ped))
+                                TriggerServerEvent('qb-vehicleshop:server:checkifFinanced')
+                                Wait(1000)
+                                if financed then
```

Comment:
Where does the variable financed get updated? You triggered a new server event that does not exist and did not even supply the contents of the new event. Also you did not use a server callback so unless you create a client side event to trigger from the server side to handle updating the financed local class variable it wont work. 

Overall this shouldnt have been a PR as it is incomplete therefore not a fix.